assets/js/editor-dark-mode-support.js

```diff
@@ -0,0 +1,6 @@
+if (
```

Comment:
It's possible this could change once the user's loaded the page (Macs have an "auto" setting which will switch to dark in the evening). I played around with `matchMedia().addListener()` locally & got it working, mind if I push that to this branch?

Reply:
Of course, feel free to push :heart: